addon/components/u-edit/shared-triggers/modals/image-upload.ts

```diff
@@ -1,6 +1,12 @@
 import UEDitFileUploader from '@upfluence/ember-upf-utils/components/u-edit/shared-triggers/modals/file-uploader';
+import { FileValidator } from '@upfluence/oss-components/addon/types/uploader';
 
 export default class UEditImageUploadComponent extends UEDitFileUploader {
-  allowedExtensions = ['jpg', 'jpeg', 'png', 'gif'].join(',');
   titleKey = 'image';
+  fileUploadRules: FileValidator[] = [
+    {
+      type: 'filetype',
+      value: ['jpg', 'jpeg', 'png', 'gif']
```

Comment:
you can use `value: ['image', 'gif']` as `image` is expanded into all these options: https://github.com/upfluence/oss-components/blob/f082728e7acc5d43ae7e8b06f57c7fba97098218/addon/utils/uploader-validators/index.ts#L31.